source/_integrations/stream.markdown

```diff
@@ -13,48 +13,17 @@ ha_codeowners:
 ha_domain: stream
 ---
 
-The `stream` integration provides a way to proxy live streams through Home Assistant. The integration currently only supports proxying H.264 source streams to the HLS format and requires at least FFmpeg >= 4.
+The `stream` integration provides a way to proxy live streams through Home Assistant as an internal component used by the [`camera`](/integrations/camera) integration. The integration currently only supports proxying H.264 source streams to the HLS format and requires at least FFmpeg >= 4.
```

Comment:
```suggestion
The `stream` integration provides a way to proxy live streams through Home Assistant as an internal component used by the [`camera`](/integrations/camera) integration. The integration currently supports proxying H.264 and H.265 source streams to the HLS protocol and requires at least FFmpeg >= 4. Note that H.265 support is limited to Safari, iOS, and Android.
```

Reply:
Honestly, all these terms should be removed from the introduction text.
Please note, that we are getting more and more users with less technical knowledge towards our platform. Throwing in a bunch of codecs isn't going to help.

It is fine to add some clarity later on, but let us keep it out of the first paragraph.

Reply:
Sure, we can move the whole section down then. But by what you are saying, even the existing intro might be a bit too much as it mentions proxying, H.264, and the ffmpeg version requirement.
Maybe it should just read something like
"The `stream` integration is an internal component used by `camera` entities to provide video streams."